proto/services/property_svc/v1/property_value_svc.proto

```diff
@@ -2,55 +2,77 @@ syntax = "proto3";
 
 package proto.services.property_svc.v1;
 
-option go_package = "gen/proto/services/property-svc";
+import "proto/services/property_svc/v1/types.proto";
+import "google/protobuf/timestamp.proto";
 
-// TODO: This proto is just an hello-world example and far from where it should be!
+option go_package = "gen/proto/services/property-svc";
 
 //
-// CreatePropertyValue
+// AttachPropertyValueRequest
 //
-message CreatePropertyValueRequest{
-  string subject_id = 1;
-  string subject_type = 2;
-  string property_id = 3;
+message AttachPropertyValueRequest{
+  string subject_id = 1; // @gotags: validate:"uuid4"
+  string property_id = 2; // @gotags: validate:"uuid4"
+
   oneof value {
-    string text_value = 4; // FIELD_TYPE_TEXT
-    float number_value = 5; // FIELD_TYPE_NUMBER
+    string text_value = 4; // @gotags: validate:"required" // FIELD_TYPE_TEXT
```

Comment:
did you test this? or does the "required" also force a value to exist

Reply:
Yes, if the string is empty we will get an validation-error

Reply:
I sure hope so, but how does it behave when no string is provided at all?

Reply:
It allows no string as well.